### PR TITLE
Improvements for the drawer header layout

### PR DESCRIPTION
**Pull request by mauriciocolli**
I noticed that some views are misaligned, specially when the app name is not the short one.

- Align views in the drawer header.
- Enable autosizing in the text view showing the app name.
- Use a proper drop up/down arrow icon.
- Move selected service views for better visibility.
- Show the selected service icon next to its name.
- Add a subtle fade background to the service name/icon for better readability.

### Visual differences

| Before  | After |
|-|-|
|  |  |
|  |  |

Autosizing doing its work when the app name is longer than usual:

##### Also, what do you prefer, [central](https://user-images.githubusercontent.com/13389907/77258264-5360dc00-6c58-11ea-9294-587ceb5243a1.png) or [right aligned](https://user-images.githubusercontent.com/13389907/77257251-e8aca200-6c51-11ea-83fd-b6597f1e5391.png) service name/icon views?

---

I also noticed a minor bug related to the focus: when you press on the drawer header, due to how the menu items are removed/added, the focus just get transfered to another view (currently, the one that is below the drawer), causing an interruption in the animation of the pressed view.

It messes up the navigation using anything other than touch too (e.g. keyboard).

Won't fix it in this pull request though.

### CODE REVIEW

**Comment by gkeegan** [comment]
I think the right-aligned service names looks better and will be more consistent making the arrow be in the same spot for each service selected.

**Comment by TobiGr** [comment]
Nice. I prefer the centred service name.

**Comment by mauriciocolli** [comment]
Let's see then, use the reactions to vote for your favorite:
  
| :+1: → Right | :heart: → Centered | :rocket: → Balanced |
|-|-|-|
|  |  |  |



###### Open the images to see it better.

File: app/src/main/java/org/schabi/newpipe/MainActivity.java [hunk 1]
```diff
@@ -370,7 +369,7 @@ public void onNothingSelected(AdapterView<?> parent) {
     }
 
     private void showTabs() throws ExtractionException {
-        serviceArrow.setImageResource(R.drawable.ic_arrow_down_white);
+        serviceArrow.setImageResource(R.drawable.ic_arrow_drop_down_white_24dp);
```
Comment (TobiGr) [inline app/src/main/java/org/schabi/newpipe/MainActivity.java]: Same here.

File: app/src/main/java/org/schabi/newpipe/MainActivity.java [hunk 2]
```diff
@@ -312,7 +311,7 @@ private void toggleServices() {
     }
 
     private void showServices() {
-        serviceArrow.setImageResource(R.drawable.ic_arrow_up_white);
+        serviceArrow.setImageResource(R.drawable.ic_arrow_drop_up_white_24dp);
```
[the remark this answers is not in the document]
Comment (Stypox) [reply app/src/main/java/org/schabi/newpipe/MainActivity.java]: I think it is used in the description dropdown

[the remark this answers is not in the document]
Comment (mauriciocolli) [reply app/src/main/java/org/schabi/newpipe/MainActivity.java]: Done.

@Stypox that's another one, will be replaced in #3271 too.

**Pull request merged**